index.html

```diff
@@ -64,104 +64,109 @@ <h2>I’am a Software Developer</h2>
+              </p>
+              <ul class="list-container">
+                <li class="list-2"><a href="">html</a></li>
+                <li class="list-2"><a href="">bootstrap</a></li>
+                <li class="list-2"><a href="">Ruby</a></li>
+              </ul>
+              <button class="base-btn">See Project</button>
+            </article>
+            <article class="bcg-container project-card card-3 cd">
+              <h3>Website Protfolio</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="list-container">
+                <li class="list-2"><a href="">html</a></li>
+                <li class="list-2"><a href="">bootstrap</a></li>
+                <li class="list-2"><a href="">Ruby</a></li>
+              </ul>
+              <button class="base-btn">See Project</button>
+            </article>
+            <article class="bcg-container project-card card-4 cd">
+              <h3>Profesional Art Printing Data More</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="list-container">
+                <li class="list-2"><a href="">html</a></li>
+                <li class="list-2"><a href="">bootstrap</a></li>
+                <li class="list-2"><a href="">Ruby</a></li>
+              </ul>
+              <button class="base-btn">See Project</button>
+            </article>
+            <article class="bcg-container project-card card-5 cd">
+              <h3>Data Dashboard Healthcare</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="list-container">
+                <li class="list-2"><a href="">html</a></li>
+                <li class="list-2"><a href="">bootstrap</a></li>
+                <li class="list-2"><a href="">Ruby</a></li>
+              </ul>
+              <button class="base-btn">See Project</button>
+            </article>
+            <article class="bcg-container project-card card-6 cd">
+              <h3>Website Protfolio</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="list-container">
+                <li class="list-2"><a href="">html</a></li>
+                <li class="list-2"><a href="">bootstrap</a></li>
+                <li class="list-2"><a href="">Ruby</a></li>
+              </ul>
+              <button class="base-btn">See Project</button>
+            </article>
+        </div>
```

Comment:
- [ ] Kindly fix the height of the project cards to match the Figma design.
- [ ] Please adjust the display of the "See Project" button in desktop view to match the Figma design. (You can make it so that the button only appears when the card is hovered over and the button has to be inside of the card)

| You design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/159978843-c3d30d4d-f464-44a3-9a39-4fd8704c5426.png) | ![image](https://user-images.githubusercontent.com/62290545/159978697-072e19a6-681c-4b24-ad65-25f22321ecc5.png) |

Reply:
I have fixed the height and the button's position to fit the parent container.